config/packages.yml

```
@@ -88,17 +92,13 @@ acquia/blt:
   type: composer-plugin
   core_matrix:
     '>=9.0.0':
-      version: 12.x
+      version: 13.x
+      # Yeah I know, it's weird that the dev version is lower than stable. It's because of a stale branch alias.
```

Lol. That's funny, but it's written in a different voice from the rest of the codebase. 🙂  And I think we can clarify the terms a little. (They confused me at first.)
```suggestion
      # It doesn't seem right, but "dev_version" must use a lower major version branch than "version" due to a stale branch alias.
```